cicd/pipelines/securing-webhooks-with-event-listeners.adoc

```diff
@@ -0,0 +1,22 @@
+[id="securing-webhooks-with-event-listeners"]
+= Securing webhooks with event listeners
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+:context: securing-webhooks-with-event-listeners
+
+toc::[]
+
+Triggers in {pipelines-title} support both HTTP (insecure) and HTTPS (secure HTTP) connections to the `Eventlistener` resource. With the secure HTTPS connection, you get an end-to-end secure connection within and outside the cluster. After you create a namespace, you can enable this secure HTTPS connection for the `Eventlistener` resource by adding the `operator.tekton.dev/enable-annotation=enabled` label to the namespace, and then creating a `Trigger` resource and a secured route using re-encrypt TLS termination.
```

Comment:
> After you create a namespace, you can enable this secure HTTPS connection for the `Eventlistener` resource by adding the `operator.tekton.dev/enable-annotation=enabled` label to the namespace, and then creating a `Trigger` resource and a secured route using re-encrypt TLS termination.

Too long sentence. Consider breaking down, use active voice, and establish a clear narrative.